src/lib.rs

```diff
@@ -498,6 +498,7 @@ use thiserror::Error;
 
 pub use ice::IceConnectionState;
 
+pub use dtls::Fingerprint;
```

Comment:
I will move this eventually since it's low level API use, and I don't want that top level in the crate (again for teachability). Later problem though since I don't have a clear idea of where and what to group together.

Reply:
I leave these open for you to move.